docs/en/ingest-management/beats-agent-comparison.asciidoc

```diff
@@ -238,6 +240,106 @@ per output type, which enables loadbalancing.
 |Supported
 |===
 
+[discrete]
+[[additional-capabilties-beats-and-agent]]
+== Additional capabilities
+
+The following table shows the additional capabilities supported by {beats} and the {agent} in {version}:
+
+
+[options,header]
+|===
+|Item |{beats} |{fleet}-managed {agent} |Standalone {agent}
+
+|Single agent for all use cases
+|{n}
+|{y}
+|{y}
+
+|Install integrations from Kibana UI or API
+|{n}
+|{y}
+|{y}
+
+|Configure from Integrations UI
+|{n}
+|{y}
+|{y} (optional)
+
+|Central, remote agent policy management
+|{n}
+|{y}
+|{n}
+
+|Central, remote agent binary upgrades
+|{n}
+|{y}
+|{n}
+
+|Choose which module or integration assets to add
+|{n}
+|{y}
+|{y}
+
+|Auto-generated {es} API keys
+|{n}
+|{y}
+|{n}
+
+|Auto-generate minimal {es} privileges on the edge
+|{n}
+|{y}
+|{y} (when using Integrations UI)
+
+|Data streams support
+|{n}
+|{y}
+|{y}
+
+|Variables and input conditions
+|{n}
+|{y}
+|{y}
+
+|Requires superuser role to add integrations and agent polices
+|{n}
+|{y} 
+|{n} (it's optional to use the Integrations UI)
+
+|Air-gapped network support
```

Comment:
Call out the specific components that need internet connection. For EPR, point to package registry repo on how to run offline EPR as an experimental feature and how to connect integrations UI to it.

Reply:
done